scripts/keras_benchmarks/models/cifar10_cnn_benchmark.py

```diff
@@ -0,0 +1,96 @@
+'''
+Original Model from keras/examples/cifar10_cnn.py
+
+Benchmark CNN model
+'''
+
+from __future__ import print_function
+import time
+import numpy as np
+import keras
+from keras.models import Sequential
+from keras.layers import Dense, Dropout, Activation, Flatten
+from keras.layers import Conv2D, MaxPooling2D
+from keras.utils import multi_gpu_model
+
+from models import timehistory
+from data_generator import generate_img_input_data
+if keras.backend.backend() == 'cntk':
+    from gpu_mode import cntk_gpu_mode_config
+
+
+class Cifar10CnnBenchmark():
+
+    def __init__(self):
+        self.test_name = "cifar10_cnn"
+        self.sample_type = "images"
+        self.total_time = 0
+        self.batch_size = 32
+        self.epochs = 2
+        self.num_samples = 1000
+
+    def run_benchmark(self, gpus=0):
+        num_classes = 10
+
+        # Generate random input data
+        input_shape = (self.num_samples, 3, 32, 32)
+        x_train, y_train = generate_img_input_data(input_shape)
+
+        y_train = np.reshape(y_train, (len(y_train), 1))
+        y_train = keras.utils.to_categorical(y_train, 10)
+
+        if keras.backend.image_data_format() == 'channels_last':
+            x_train = x_train.transpose(0, 2, 3, 1)
+
+        model = Sequential()
+        model.add(Conv2D(32, (3, 3), padding='same',
+                         input_shape=x_train.shape[1:]))
+        model.add(Activation('relu'))
```

Comment:
You can also remove the `Activation` layers in this model and incorporate them into the previous layer.